### ISSUE TITLE
Equation block - throw er; // Unhandled 'error' event

### ISSUE THREAD

**Issue by lucasmelin** [idyll-lang/idyll#239]
Attempting to compile a .idl file containing an equation block produces the following error:

```
throw er; // Unhandled 'error' event
Error: Cannot find module './src/Settings' from 'C:\Users\Lucas\AppData\Roaming\npm\node_modules\idyll\node_modules\katex\dist'
    at C:\Users\Lucas\AppData\Roaming\npm\node_modules\idyll\node_modules\browser-resolve\node_modules\resolve\lib\async.js:55:21
    at load (C:\Users\Lucas\AppData\Roaming\npm\node_modules\idyll\node_modules\browser-resolve\node_modules\resolve\lib\async.js:69:43)
    at onex (C:\Users\Lucas\AppData\Roaming\npm\node_modules\idyll\node_modules\browser-resolve\node_modules\resolve\lib\async.js:92:31)
    at C:\Users\Lucas\AppData\Roaming\npm\node_modules\idyll\node_modules\browser-resolve\node_modules\resolve\lib\async.js:22:47
    at FSReqWrap.oncomplete (fs.js:166:21)
```

**Comment by lucasmelin** [comment]
Seems like this could be a Katex issue instead, https://stackoverflow.com/questions/48045838/require-npm-module-with-browserify-cannot-find-dependency

**Comment by lucasmelin** [comment]
Based on reading https://github.com/staltz/mmmmm-mobile/issues/1 I've managed to get this working by deleting the node_modules folder from npm\node_modules\idyll and then from the idyll directory, running yarn.

**Comment by mathisonian** [comment]
This should be fixed in the latest version `v2.1.4`, just released on npm
